src/lerobot/scripts/lerobot_train.py

```diff
@@ -88,38 +84,35 @@ def update_policy(
         - A dictionary of outputs from the policy's forward pass, for logging purposes.
     """
     start_time = time.perf_counter()
-    device = get_device_from_parameters(policy)
     policy.train()
-    with torch.autocast(device_type=device.type) if use_amp else nullcontext():
+
+    # Let accelerator handle mixed precision
+    with accelerator.autocast():
         loss, output_dict = policy.forward(batch)
         # TODO(rcadene): policy.unnormalize_outputs(out_dict)
-    grad_scaler.scale(loss).backward()
 
-    # Unscale the gradient of the optimizer's assigned params in-place **prior to gradient clipping**.
-    grad_scaler.unscale_(optimizer)
+    # Use accelerator's backward method
+    accelerator.backward(loss)
 
-    grad_norm = torch.nn.utils.clip_grad_norm_(
-        policy.parameters(),
-        grad_clip_norm,
-        error_if_nonfinite=False,
-    )
+    # Clip gradients if specified
+    if grad_clip_norm > 0:
+        grad_norm = accelerator.clip_grad_norm_(policy.parameters(), grad_clip_norm)
+    else:
+        grad_norm = torch.tensor(0.0, device=accelerator.device)
```

Comment:
Why did you set grad_norm here to zero? It's a very important metric for debugging you can just keep it as before if its not set
```suggestion
        grad_norm = torch.nn.utils.clip_grad_norm_(
            policy.parameters(), float('inf'), error_if_nonfinite=False
        )
```
or just calculate the norm directly
```
        grad_norm = torch.norm(torch.stack([p.grad.norm(2) for p in policy.parameters() if p.grad is not None]), 2)
```